kong/plugins/jwt/handler.lua

```diff
@@ -28,7 +28,13 @@ local function retrieve_token(request, conf)
     end
   end
 
-  local authorization_header = request.get_headers()["authorization"]
+  local jwt_key = "authorization"
+  local authorization_header
+  authorization_header = ngx.unescape_uri(ngx.var["cookie_" .. jwt_key])
```

Comment:
isn't the JWT token url-safe? so why would it need to be unescaped?

Reply:
Yes, it must be unescaped because jwt uses a cookie to send an escape, and the Bearer xx is escaped to be Bearer+xx

Reply:
ok, but why would we include the "bearer " prefix to the cookie contents? It forces a url-decode on the entire string. If we drop the prefix, just store the jwt token, we can skip the urldecode, right?

Reply:
Yes, I am in order to keep consistent with the header way, if you do not need to keep the same, then I have to adjust the next, how do you think？

Reply:
i removed the jwt 'bearer ' prefix from cookie